src/datascience-ui/variable-view/variableViewPanel.tsx

```diff
@@ -50,12 +59,27 @@ export class VariableViewPanel extends React.Component<IVariableViewPanelProps>
         // Return our variable panel, we wrap this in one more top level element "variable-view-main-panel" so that
         // we can size and host it differently from the variable panel in the interactive window or native editor
         return (
-            <div id="variable-view-main-panel" role="Main" style={dynamicFont}>
+            <div id="variable-view-main-panel" role="Main" ref={this.panelRef} style={dynamicFont}>
                 {this.renderVariablePanel(this.props.baseTheme)}
             </div>
-        ); // NOTE: Currently the OPEN, button just exists to mimic the toggling of the variable view, make it easier to test when working
+        );
     }
 
+    private windowResized = () => {
```

Comment:
Same resize / timer code that we used in other spots reactSlickGrid / monacoEditor.